cmd/root.go

```diff
@@ -129,6 +129,18 @@ func AddCommands(cmd *cobra.Command) {
 	cmd.PersistentFlags().IntVar(&config.HARE.ExpectedLeaders, "hare-exp-leaders",
 		config.HARE.ExpectedLeaders, "The expected number of leaders in the hare protocol")
 
+	/**======================== Hare Eligibility Oracle Flags ========================== **/
+
+	// N determines the size of the hare committee
+	cmd.PersistentFlags().Uint64Var(&config.ELIGIBILITY.ConfidenceInterval, "eligibility-confidence-interval",
+		config.ELIGIBILITY.ConfidenceInterval, "The confidence interval")
+	// F determines the max number of adversaries in the Hare committee
```

Comment:
if you are trying to be consistent with the Spacemesh paper, then 'F' should be 'f'.

Reply:
For public only